### PR TITLE
tidb-server: Fix issue #371

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: tidb-server/server/util_test.go [hunk 1]
```diff
@@ -15,9 +15,40 @@ package server
...
 }
+
+func (s *testUtilSuite) TestDumpBinaryTime(c *C) {
+	tests := []struct {
+		Input  string
+		Expect []byte
+	}{
+		{"0000-00-00 00:00:00.0000000", []byte{11, 1, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0}},
+		{"0000-00-00", []byte{4, 1, 0, 1, 1}},
+		{"0000-00-00 00:00:00.0000000", []byte{0}},
```
Comment (shenli) [inline tidb-server/server/util_test.go]: Why all the testcases are zeros?

Comment (zimulala) [inline tidb-server/server/util_test.go]: fix #371
I will add more tests.

Comment (shenli) [inline tidb-server/server/util_test.go]: If your pr only handle zerotime. These test cases are enough.

Comment (zimulala) [inline tidb-server/server/util_test.go]: got it.

Comment (qiuyesuifeng) [inline tidb-server/server/util_test.go]: It is a little confused to use tests variables, for different item will do different test check?
I think define separate variables it better.

Comment (zimulala) [inline tidb-server/server/util_test.go]: gotcha.

**Comment by shenli** [comment]
LGTM

**Pull request merged**